.coveragerc

```diff
@@ -28,3 +28,6 @@ source_pkgs =
 omit =
     mysite/asgi.py
     mysite/wsgi.py
+plugins =
+  django_coverage_plugin
+  covdefaults
```

Comment:
Adding this plug-in should allow you to remove the `exclude_also` entry, and maybe some others. Check the docs.

Reply:
> Adding this plug-in should allow you to remove the `exclude_also` entry, and maybe some others. Check the docs.

Yes, thank you. I deleted all, that were defined in the defaults setting

Reply:
Marking as resolved.